docker/envoy_mixer.json

```diff
@@ -0,0 +1,44 @@
+{
+  "listeners": [
+   {
+    "address": "tcp://0.0.0.0:15004",
+    "name": "tcp_0.0.0.0_15004",
+    "filters": [
+     {
+      "type": "read",
+      "name": "tcp_proxy",
+      "config": {
+       "stat_prefix": "tcp",
```

Comment:
why is this just tcp proxy? we should have http_connection_manager here. Or is the goal to just secure the channel?

Reply:
For now the goal is to secure the channel. Will send out a  different PR for getting it to http connection manager.